src/fps/FPS.md

```diff
@@ -48,4 +48,33 @@ isNestedSafe = false
 
 ### Task Template Files
 
-TBD, will come in another PR
+Task templates allow task developers to create new tasks without writing any Solidity code. The templates are designed to be as flexible as possible, allowing developers to create tasks that update multiple parameters for a single or superchain. Templates are currently in development and will be released in a future PR.
```

Comment:
Can we change the name of this file to something other than FPS.md? Is that outdated now?